packages/db/prisma/schema.prisma

```diff
@@ -74,15 +74,26 @@ model BlobStoragesState {
   @@map("blob_storages_state")
 }
 
-model Address {
-  address                    String   @id
-  firstBlockNumberAsSender   Int?     @map("first_block_number_as_sender")
+model AddressHistory {
```

Comment:
I don't like the name tbh. How about naming it AddressData, AddressCategoryInfo or AddressByCategoryInfo? :thinking:

Reply:
I like `AddressCategoryInfo`